scenes/game_elements/characters/enemies/guard/guard.tscn

```diff
@@ -5,7 +5,7 @@
 [ext_resource type="Texture2D" uid="uid://8t6ihmesvghq" path="res://scenes/game_elements/characters/enemies/guard/components/field_of_view.png" id="3_0hjcv"]
 [ext_resource type="Texture2D" uid="uid://c75ofhy3swhj3" path="res://scenes/game_elements/characters/enemies/guard/components/exclamation_mark.png" id="3_pnkgp"]
 [ext_resource type="Script" uid="uid://cxsi2xqcdyw7g" path="res://scenes/game_elements/characters/enemies/guard/components/light.gd" id="4_lptvm"]
-[ext_resource type="SpriteFrames" uid="uid://ovu5wqo15s5g" path="res://scenes/quests/story_quests/template/components/guard_enemy_template.tres" id="5_mswbt"]
+[ext_resource type="SpriteFrames" uid="uid://ovu5wqo15s5g" path="res://scenes/quests/story_quests/template/template_components/template_guard_enemy.tres" id="5_mswbt"]
```

Comment:
Have you agreed to also rename the `components/` folder to `template_components/` ?

Reply:
That was discussed briefly & agreed, though we were in a bit of a hurry and it was kind of an arbitrary choice.

Reply:
OK! Then let's merge.